src/Orleans.Runtime/MembershipService/ClusterMembershipService.cs

```diff
@@ -99,7 +99,7 @@ await foreach (var tableSnapshot in this.membershipTableManager.MembershipTableU
         void ILifecycleParticipant<ISiloLifecycle>.Participate(ISiloLifecycle lifecycle)
         {
             var tasks = new List<Task>(1);
-            var cancellation = new CancellationTokenSource();
+            using var cancellation = new CancellationTokenSource();
```

Comment:
```suggestion
            var cancellation = new CancellationTokenSource();
```

This cancellation doesn't need to be disposed - it is used to signal shutdown to the `ProcessMembershipUpdates` loop, but doesn't have a timer or anything else associated with it which would benefit from disposal.